contents/docs/cdp/batch-exports/index.mdx

```diff
@@ -79,7 +86,7 @@ More information, including any additional necessary permissions, schema informa
 * [Postgres](/docs/cdp/batch-exports/postgres)
 * [Redshift](/docs/cdp/batch-exports/redshift)
 
-## Sessions model <span class="bg-gray-accent-light dark:bg-gray-accent-dark text-gray font-semibold align-middle text-sm p-1 rounded ml-1">Beta</span>
+### Sessions model
```

Comment:
adding this additional `<span>` was breaking automatic scroll when clicking the link in the sidebar